src/gui/wizard/owncloudwizard.cpp

```diff
@@ -302,6 +376,34 @@ void OwncloudWizard::changeEvent(QEvent *e)
 void OwncloudWizard::customizeStyle()
 {
     // HINT: Customize wizard's own style here, if necessary in the future (Dark-/Light-Mode switching)
+    auto theme = Theme::instance();
+    auto wizardPalette = palette();
+
+    wizardPalette.setColor(QPalette::Window, theme->wizardHeaderBackgroundColor());
+
+    wizardPalette.setColor(QPalette::Base, theme->wizardHeaderBackgroundColor());
+    QColor colorBaseDisabled(wizardPalette.color(QPalette::Base));
+    colorBaseDisabled.setAlpha(128);
```

Comment:
@FlexW Since we have **128** repeated 3 times, can we have it as a `const` variable instead?
`const int colorDisabledAlpha = 128`